resources/views/layouts/default/widgets/live_map.blade.php

```diff
@@ -64,6 +64,7 @@
   </div>
 </div>
 
+@if($config['table'] == 'on')
```

Comment:
`@if($config['table'] === true)`

Reply:
Why ? Specially where everything else in the config part was pure strings, what was the reason for having the boolen for the table display part.

I do understand making all in same type where possible without breaking the code flow but for this nothing was wrong by having it as string. This is just your personal wish i suppose while my personal wish was having it as a string like others :) It works either way and you are free to edit any pr's I sent, you do not need to wait me for the edits you think are needed.